src/styles/blocks/buy.scss

```diff
@@ -0,0 +1,20 @@
+.card__buy-button {
```

Comment:
The size of the buttons are not corresponding to the referene because your trying to define the size of the button by just using padding. It won't work in this case. It's better to use the height and width properties to define the size.